File: projekatN/src/main/java/com/example/model/Address.java
package com.example.model;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Entity
public class Address {

	@Id
	@GeneratedValue
	private Long id;

	private String street;
	private String number;
	private int zipCode;
	private String city;

	public Address() {

	}

	public Address(Long id, String street, String number, int zipCode, String city) {
		super();
		this.id = id;
		this.street = street;
		this.number = number;
		this.zipCode = zipCode;
		this.city = city;
	}

	public Address(String street, String number, int zipCode, String city) {
		super();
		this.street = street;
		this.number = number;
		this.zipCode = zipCode;
		this.city = city;
	}

	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public String getStreet() {
		return street;
	}

	public void setStreet(String street) {
		this.street = street;
	}

	public String getNumber() {
		return number;
	}

	public void setNumber(String number) {
		this.number = number;
	}

	public int getZipCode() {
		return zipCode;
	}

	public void setZipCode(int zipCode) {
		this.zipCode = zipCode;
	}

	public String getCity() {
		return city;
	}

	public void setCity(String city) {
		this.city = city;
	}

}

File: frontend/src/main/frontend/src/app/models/address.ts
export class Address implements AddressInterface {
	public id: number;
	public street: string;
	public number: string;
    public zipCode: number;
    public city: string;
		
	constructor(addressCfg:AddressInterface)
	{	
		this.id = addressCfg.id;
		this.street = addressCfg.street;
		this.number = addressCfg.number;
        this.zipCode = addressCfg.zipCode;
        this.city = addressCfg.city;
	}
}

interface AddressInterface{
	id?: number;
    street: string;
    number: string;
    zipCode: number;
    city: string;
}